components/Main/index.js

```diff
@@ -1,22 +1,155 @@
-import { getLocalStorage } from "../../helpers/localStorage.js";
-import { getProjectFromLocalStorageById } from "../../services/project.js";
+import {
+  getLocalStorage,
+  setLocalStorage,
+} from "../../helpers/localStorage.js";
+import {
+  getProjectById,
+  deleteProject,
+  updateProjects,
+} from "../../services/project.js";
+import { previewProjectsNameOnTheSidebar } from "../Sidebar/index.js";
+import { renderCurrentProject } from "../Sidebar/index.js";
 let proName = document.querySelector(".project-name");
```

Comment:
`proName` is not descriptive. A better name is `projectNameElement`, as it describes the variable's purpose.